style/crawler/crawl.py

```diff
@@ -1,29 +1,130 @@
-from bs4 import BeautifulSoup
+from abc import ABC
+from abc import abstractmethod
+from os import mkdir
+from os import path
+from pathlib import Path
+
 import requests
+from bs4 import BeautifulSoup
+from urlpath import URL
+
+from utils.author_catalog import create_catalog
+from utils.utils import sanitize_author_name
+
+gutenberg_base_url = "http://aleph.gutenberg.org/"
+file_path_book_db = path.join(Path(__file__).parents[2], "book_db")
```

Comment:
Path library can provide some easy concatenation, so you don't need to use `path.join`.